contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,303 @@ availability:
 
-#### 1. Experiment name, feature flag, and testing variants
+![How to create an experiment in PostHog](../../images/features/experiments/create-experiment.png)
 
-Let's start in the upper left corner by filling out a name, description, and feature flag for our experiment. Fill in a name for your feature flag - it will be created here by default, linked to the experiment rather than existing as its own entity.
+Here's a breakdown of each field in the form:
 
-![Name, feature flag, and variants](../../images/features/experiments/create-experiment-name-variants.png)
+### Feature flag key
 
-We can also include multivariants for our testing changes. For example, instead of running our experiment with just "control" and "test", we can test for the green visual button change or the orange one or our original control.
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you'll use this feature flag key to check which experiment variant the user has been assigned to. 
 
-#### 2. Participant selection and experiment goal/metric
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](// TODO add link here) 
 
-Next, we want to select our experiment's universe. Here, we've decided that for our sign up button experiment we want to test our hypothesis on users from Asia.
+### Experiment variants
 
-We also want to set our experiment's "goal" now for which metric we want to track. Since we're testing whether the number of sign up button clicks increase or not, we'll use the "Trend" goal to measure aggregate values
+By default, all experiments have a 'control' and 'test' variant. If you have more variants that you'd like to test, you can use this field to add more variants and run multivariant tests.
 
-![Participants and goal metric](../../images/features/experiments/create-experiment-participants-goals.png)
+### Participant type
 
-If we want to test a hypothesis for a conversion rate metric, we'll use the "Funnel" goal instead.
+If you've created [groups](/docs/product-analytics/group-analytics) in PostHog, you can use this field to run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level. 
 
-#### 3. Experiment preview and minimal acceptable improvement threshold
+### Participant targeting
 
-Finally, in our experiment creation form, look to the upper right corner for our experiment's preview!
-We can see recommendations for run times and other experiment factors.
+By default, your experiment will target 100% of participants. If you'd like to target a more specific set of participants, or change rollout percentage, you'll need to do this on the feature flag that will be used by your experiment.
```

Comment:
This is a bit unclear, you can set filters on the experiment screen, but you also might need to go to the feature flag itself?

Reply:
I'll delete filters on experiments screen soon.